### DISCUSSION TITLE
Suggestion: Account repositories sort / order list

### DISCUSSION THREAD

**Discussion by cjsdiniz** [yhype/yhype-feedback#3] [Ideas]
I've one suggestion which is: in the account repository  list  order/sort the list by visitors / cloners / stars.
Thanks

**Comment by antonkomarev** [comment]
Thanks for the idea @cjsdiniz! How should it work? Choose how to sort repositories or sort them by default?
If sort by default, what should be more valuable (views, clones, stars)?

**Reply by cjsdiniz** [reply]
Hi Anton, 
I think the repositories could be sorted descending by Cloners , by default (since I believe more cloners a repository has more important is) with the option to be sorted ascending or descending by any of the types, if it is feasible, of course.
Thanks.

**Comment by andrey-helldar** [comment]
The organization page has a list of repositories that we can turn on or off.

The organization page has a list of repositories that we can turn on or off.

I suggest adding a double sorting so that the active ones are displayed at the top and then, within each group, are displayed alphabetically to make it easier to find the right one.

```sql
ORDER BY is_visible DESC, title ASC
```

I think this would turn a blind eye to the lack of an active list filter on the page itself))

**Reply by antonkomarev** [reply]
It will be easy to implement, if we will resort list only on page refresh, otherwise it will flicker. I need to try it.

**Reply by andrey-helldar** [reply]
Yes, I think once is enough when rendering a page.

Considering that the web interface has no sorting functionality at all :)

**Reply by antonkomarev** [reply]
@antonkomarev @cjsdiniz proposed sort order is implemented. But personal account repository will be on top without depending on scraping enabled flag.

Manual re-sorting is planned for the future, because it is much complex.

**Reply by andrey-helldar** [reply]
@antonkomarev thank you!

**Reply by lulunac27a** [reply]
Still no option yet to sort/order by clones/views/stars.
